lib/commands/build.js

```diff
@@ -59,12 +60,29 @@ function buildCommand(argv) {
   }
 
   // TODO: Check stripes-core location and aliases to warn if build is unsuitable for production.
+  const baseStripesConfig = platform.getStripesConfig();
+  const { okapi } = baseStripesConfig;
 
-  console.log('Building...');
-  const stripes = _stripesCore || new StripesCore(context, platform.aliases);
-  stripes.api.build(platform.getStripesConfig(), Object.assign({}, argv, { webpackOverrides }))
-    .then(processStats)
-    .catch(processError);
+  let discoveryPromise = Promise.resolve(baseStripesConfig);
+  if (okapi && okapi.url) {
+    console.log(`Fetching list of modules from ${okapi.url} for tenant ${okapi.tenant}`);
+
+    discoveryPromise = fetch(`${okapi.url}/_/proxy/tenants/${okapi.tenant}/modules?full=true`)
```

Comment:
Looks like a reasonable solution. As I'm not actively involved, I will defer to others on the build-time vs. runtime decisions.

Have you considered using the CLI's own [okapi-client](https://github.com/folio-org/stripes-cli/blob/master/lib/okapi/okapi-client.js)?  If we go with this approach, I am inclined fetch the modules via the Okapi client rather than doing so right here in the command.  The client also offers instrumentation for debugging.

For instance, `stripes mod list` already performs a similar operation (less the `full=true` option).  I could see encapsulating your full variant in [module-service](https://github.com/folio-org/stripes-cli/blob/master/lib/okapi/module-service.js), similar to [this one](https://github.com/folio-org/stripes-cli/blob/027ea3766085a526dff98c618a87214cf9f0d601/lib/okapi/module-service.js#L82-L96) used by the list command.

Its been a while since I've looked at the output of the full=true case.  If the output is different enough, it may make sense to plumb it up with its own copy of the `/modules?full=true` route.  For reference, we have [Okapi routes defined here](https://github.com/folio-org/stripes-cli/blob/027ea3766085a526dff98c618a87214cf9f0d601/lib/okapi/okapi-repository.js#L88-L119).